db/seeds/prisons/TCI-thorn-cross.yml

```diff
@@ -22,14 +22,17 @@ recurring:
   fri:
   - 1345-1545
 unbookable:
-- 2018-12-25
-- 2018-12-26
-- 2019-01-01
+- 2019-12-24
```

Comment:
This slot (24/12) needs to be removed from the unbookable section as it's also under anomalous, which is causing a conflict/error

Reply:
Amended the the PR. I've removed the 26th and 2nd because there are no social visits at Thorn Cross prison on Thursday.  Also removed 31st from unbookable because it is under anomalous.